.github/workflows/kind.yml

```diff
@@ -404,6 +404,73 @@ jobs:
         path: log.tar.gz
         retention-days: 30
 
+  test-e2e-noencap-non-default:
```

Comment:
Could the issue be convered if this is hybrid-non-default? If yes, maybe we should use hybrid as that's what we want to recommend?

Reply:
> Could the issue be convered if this is hybrid-non-default? 

Yes, it can be, but we need to make some changes to the corresponding test `TestAntreaPolicy/TestGroupNoK8sNP/Case=ACNPNodePortServiceSupport`.

The reason is that this issue can be only triggered when two Nodes are in the same subnet. For the testbed of the current Github action hybrid test (control-plane, subnet 172.18.0.0/24, worker, worker1, subnet 20.20.20.0/24), to cover the issue, we need to ensure that the NodePort ingress Node and its Endpoint Node are in the same subnet in hybrid mode in `TestAntreaPolicy/TestGroupNoK8sNP/Case=ACNPNodePortServiceSupport`. 

I'm okay with adding `test-e2e-hybrid-non-default` instead of `test-e2e-noencap-non-default`. Will make the change.